atc/creds/ssm/manager.go

```diff
@@ -1,32 +1,32 @@
 package ssm
 
 import (
+	"code.cloudfoundry.org/lager"
 	"encoding/json"
 	"errors"
-	"io/ioutil"
-	"strings"
-	"text/template"
-	"text/template/parse"
-
-	"code.cloudfoundry.org/lager"
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/aws/credentials"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/ssm"
 	"github.com/concourse/concourse/atc/creds"
+	"github.com/mitchellh/mapstructure"
+	"io/ioutil"
+	"strings"
+	"text/template"
+	"text/template/parse"
```

Comment:
well my IDE changed it automatically and I just ran `go fmt` and nothing changed 🤷‍♂️ I can rollback this if you want

Reply:
Ah, sounds like grouping imports is only implemented by [`goimports`](https://godoc.org/golang.org/x/tools/cmd/goimports) (https://goinbigdata.com/goimports-vs-gofmt/). `goimports` is actually even more commonplace than just `fmt` on save since it automatically adds/removes package imports, which is really convenient.

Just to avoid churn it'd be nice to undo this - try running `goimports` locally? (Your editor might pick it up and start using it.)

Perhaps this is something to add to `CONTRIBUTING.md` 🤔

Reply:
Fixed :) I might open a PR to add this to `CONTRIBUTING.md`